release-notes/opensearch-neural-search.release-notes-2.5.0.0.md

```diff
@@ -0,0 +1,22 @@
+## Version 2.5.0.0 Release Notes
+
+Compatible with OpenSearch 2.5.0
+
+### Enhancements
+
+* Add filter option for query type ([#88](https://github.com/opensearch-project/neural-search/pull/88))
+* Add retry mechanism for neural search inference ([#91](https://github.com/opensearch-project/neural-search/pull/91))
+* Enable core branching strategy and make Neural Plugin as extensible plugin. ([#87](https://github.com/opensearch-project/neural-search/pull/87))
```

Comment:
Should we break these into 2 lines one for core branching and one for Extensible plugin?